src/main/java/MathematicalCalculations.java

```diff
@@ -1,16 +1,13 @@
 import Exceptions.DivisionByZero;
 
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Stack;
 
 public class MathematicalCalculations {
-    public static double Calculations() throws DivisionByZero {
-        String reversePolandExpression = ExpressionParser.parseExpressionInReversePolandNotation(WorkWithTheConsole.getExpression());
-        ArrayList<String> numbersOrArithmeticsOperations = new ArrayList<>(Arrays.asList(reversePolandExpression.split(" ")));
-        numbersOrArithmeticsOperations.removeIf(item -> item == null || "".equals(item));
+    public static double Calculations(ArrayList<String> expression) throws DivisionByZero {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.